src/R/Components/Impl/ConnectionManager/Implementation/ConnectionManager.cs

```diff
@@ -64,6 +64,12 @@ internal class ConnectionManager : IConnectionManager {
             _userConnections = new ConcurrentDictionary<Uri, IConnection>(userConnections);
 
             UpdateRecentConnections();
+
+            _shell.Idle += OnIdle;
+        }
+
+        private void OnIdle(object sender, EventArgs e) {
```

Comment:
It looks like you have setup debugger to stop on NRE (which isn't a default). But we can check `Microsoft.VisualStudio.PlatformUI.Shell.ViewManager.Instance.WindowProfile` for null instead. 

---
In reply to: [78464734](https://github.com/Microsoft/RTVS/pull/2296#discussion_r78464734) [](ancestors = 78464734)